infra-as-code/bicep/vmss.bicep

```diff
@@ -60,16 +60,16 @@ param vmssFrontendApplicationSecurityGroupName string
 @description('The name of the backend Application Security Group.')
 param vmssBackendApplicationSecurityGroupName string
 
-@description('The Azure Active Directory group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
+@description('The Microsoft Entra group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
```

Comment:
oversight?

```suggestion
@description('The Microsoft Entra ID group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
```

Reply:
According to the product docs, it's "Microsoft Entra group" or "Microsoft Entra user"

So, if I have "ID" in another place... I need to fix it there.